tests/dedupe_unit_test.py

```diff
@@ -24,8 +24,8 @@ def setUp(self):
     def check_nb_duplicates(self, expected_number_of_duplicates):
         print("checking duplicates...")
         retries = 0
-        for i in range(0, 3):
-            time.sleep(5)  # wait bit for celery dedupe task which can be slow on travis
+        for i in range(0, 6):
+            time.sleep(15)  # wait bit for celery dedupe task which can be slow on travis
```

Comment:
Maybe it is better to add the sleep at the end of the loop, so we have to wait only if celery is to slow? Furthermore I think 10s should be enough and if not, its better to increase the counter.

Reply:
I don't think it really matters, but I have changed it so now you can't delay approval ;-)